link-verifier/trusted_certs/ca_bundle.crt

```diff
@@ -0,0 +1,90 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Is it possible to add some comments to describe which URL these certificates are used for?

Reply:
Done.